Automattic/vip-go-mu-plugins
# Search Dev Tools: bump Prism.js version bump to 1.25.0

PR #2440 prompted upgrade due to security concerns (https://github.com/Automattic/vip-go-mu-plugins/pull/2440)

## Description
In #2440 we merged the updated version of Prism.js for Search Dev Tools. This PR includes the build artifacts relevant to that change.

## Changelog Description

### Plugin updated: Enterprise Search

We have updated Enterprise Search Dev Tools to use the latest version of Prism.js syntax highlighting for queries.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out.
1. Perform a search
1. Click on VIP dev tools in admin bar
1. Inspect search results. make sure syntax highlighting is correct on JSON payloads..
